account_credit_control/tests/test_credit_control_run.py

```diff
@@ -249,6 +249,7 @@ def test_sent_email_invoice_detail(self):
         self.assertIn(self.invoice.name, new_communication.message_ids.body)
         # CASE 2: set the policy level to show invoice details = False
         control_lines.policy_level_id.mail_show_invoice_detail = False
+        control_lines.state = "to_be_sent"
```

Comment:
Why this change is needed?

Reply:
Because after send the emails I chage the state from queued to sent
https://github.com/OCA/credit-control/pull/430/files#diff-ab2ca28045e4ca81ddb0cacf900797263a04da3c88c8f3f46b93ebeca3b2d9bbR261

The original code not change the state!!